### ISSUE TITLE
E2E Tests Timeout Due to Promises not resolving

### ISSUE THREAD

**Issue by rileyajones** [ampproject/amphtml#35405]
### Description

A handful of tests have been flaking fairly consistently and when running the tests locally it seems that the test executes and then "passes" but does not complete and thus times out triggering a failure.

![image](https://user-images.githubusercontent.com/78179109/127057274-406f22b4-7a16-4f60-a88e-c4885bbf8b3b.png)

### Reproduction Steps

Run one of the effected tests locally (potentially more than once) and note that they fail
`amp e2e --nobuild  --compiled --files=extensions/amp-accordion/1.0/test-e2e/test-amp-accordion.js`

### Relevant Logs

```shell
https://app.circleci.com/pipelines/github/ampproject/amphtml/13626/workflows/103f55cc-ae29-4541-868b-07802e4d53e8/jobs/222868/steps

[14:56:06] Running tests...
●●●●●●●●●●●●●●●●●●●●●●●●●●●○●●○○○○○○●●●●●●●●●●●○●●○●○●●●●●●●●●●●○●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●
●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●
  1) amp-fit-text
       chrome
          Standalone environment 
           "before each" hook for "should render in correct font-size":
     Error: Server terminated early with status 1
      at /home/circleci/project/build-system/tasks/e2e/node_modules/selenium-webdriver/remote/index.js:248:24
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:95:5)

  2) amp-fit-text
       chrome
          Standalone environment 
           "after each" hook for "should render in correct font-size":
     Error: Server terminated early with status 1
      at /home/circleci/project/build-system/tasks/e2e/node_modules/selenium-webdriver/remote/index.js:248:24
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:95:5)

Executed 212 of 221 (Skipped 11) 2 FAILED
[14:57:08] Stopped server at http://0.0.0.0:8000
[14:57:08] Finished 'e2e' after 1m 10s
[14:57:08] ERROR: End-to-end tests failed!




https://app.circleci.com/pipelines/github/ampproject/amphtml/13625/workflows/c084b526-aa0a-4110-8680-b375b8af51d6/jobs/222802/steps

[14:42:30] Running tests...
●●●●●●●●●●●●●●●●●●●●●●●●●●○●●○○○○○○●●●●●●●●●●○●●○●○●●●●●●●●●●●○●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●
●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●
  1) [v1.0] amp-accordion
       chrome
          Standalone environment 
           "before each" hook for "renders section in expanded state when expanded attribute provided":
     Error: Server terminated early with status 1
      at /home/circleci/project/build-system/tasks/e2e/node_modules/selenium-webdriver/remote/index.js:248:24
      at processTicksAndRejections (internal/process/task_queues.js:95:5)

  2) [v1.0] amp-accordion
       chrome
          Standalone environment 
           "after each" hook for "renders section in expanded state when expanded attribute provided":
     Error: Server terminated early with status 1
      at /home/circleci/project/build-system/tasks/e2e/node_modules/selenium-webdriver/remote/index.js:248:24
      at processTicksAndRejections (internal/process/task_queues.js:95:5)

  3) [v1.0] amp-accordion
       chrome
          Viewer environment 
           "before each" hook for "expands and collapses when a header section is clicked":
     Error: Server terminated early with status 1
      at /home/circleci/project/build-system/tasks/e2e/node_modules/selenium-webdriver/remote/index.js:248:24
      at processTicksAndRejections (internal/process/task_queues.js:95:5)

  4) [v1.0] amp-accordion
       chrome
          Viewer environment 
           "after each" hook for "expands and collapses when a header section is clicked":
     Error: Server terminated early with status 1
      at /home/circleci/project/build-system/tasks/e2e/node_modules/selenium-webdriver/remote/index.js:248:24
      at processTicksAndRejections (internal/process/task_queues.js:95:5)

Executed 210 of 217 (Skipped 11) 4 FAILED
[14:43:34] Stopped server at http://0.0.0.0:8000
[14:43:34] Finished 'e2e' after 1m 10s
[14:43:34] ERROR: End-to-end tests failed!
```

**Comment by caroqliu** [comment]
I was having issues contributing e2e tests in #35394 due to similar "before each" hook errors on local runs. Sounds like that PR should be closed until this issue is resolved?

**Comment by rileyajones** [comment]
This seems to have been resolved by #35672 and #35671
